leechcore_device_microvmi/leechcore_device_microvmi.c

```diff
@@ -0,0 +1,71 @@
+#include <stdbool.h>
+
+#include <leechcore_device.h>
+#include <libmicrovmi.h>
+
+#define PLUGIN_URL_SCHEME "microvmi"
+
+static VOID DeviceMicrovmi_ReadContigious(PLC_READ_CONTIGIOUS_CONTEXT ctxRC)
+{
+  // read contigious physical memory
+  PLC_CONTEXT ctxLC = ctxRC->ctxLC;
+  void* driver = ctxLC->hDevice;
```

Comment:
I'd rather keep a `void*` here, because `HANDLE` is from LeechCore interface, and I'm passing this variable to libmicrovmi interface, which accepts a `void*`, not a `HANDLE`